lib/generators/decidim/install_generator.rb

```diff
@@ -118,8 +118,8 @@ def letter_opener_web
       def recreate_db
         rails_command "db:environment:set db:drop" unless ENV["CI"]
         rails_command "db:create"
-        rails_command "db:migrate"
-        rails_command "db:test:prepare"
+        rails_command "db:migrate", env: "development"
+        rails_command "db:migrate", env: "test"
```

Comment:
`rake db:test:prepare` loads the DB schema to the test DB, migrating will take longer :/